docs/api.md

```diff
@@ -690,8 +690,9 @@ iterations           | int          | yes      | N/A     | Max number of solver'
 snapshot             | int          | yes      | N/A     | Iterations between model snapshots
 snapshot_prefix      | string       | yes      | empty   | Prefix to snapshot file, supports repository
 solver_type          | string       | yes      | SGD     | from "SGD", "ADAGRAD", "NESTEROV", "RMSPROP", "ADADELTA", "ADAM",  "AMSGRAD", "RANGER", "RANGER_PLUS", "ADAMW", "SGDW", "AMSGRADW" (*W version for decoupled weight decay, RANGER_PLUS is ranger + adabelief + centralized_gradient)
-clip                 | bool         | yes      | false (true if RANGER* selected) | clip gradients, implemented only in ranger
-clip_value           | real         | yes      | 5.0     | value for  clipping gradients (used only by RANGER)
+clip                 | bool         | yes      | false (true if RANGER* selected) | clip gradients
+clip_value           | real         | yes      | 5.0     | value for  clipping gradients
```

Comment:
`gradients with absolute value greater than clip_value will be clipped` is more explicit and better imo

Same for `clip_norm`

Reply:
ok done